src/index.ts

```diff
@@ -177,16 +186,13 @@ const config = Config.fromFile("./config.toml");
     );
   }, pollIntervalMillis);
 
-  try {
-    if (bitcoinWallet) {
-      await bitcoinWallet.refreshUtxo();
-      setInterval(() => {
-        bitcoinWallet.refreshUtxo();
-      }, 60000);
-    }
-  } catch (e) {
-    logger.error("Failed to refresh UTXO:", e.message);
-    console.log(e.message);
+  if (bitcoinWallet) {
+    setTimeout(async () => {
```

Comment:
Can you explain why you are doing both please?

Reply:
otherwise this call blocks and API is never bound, by doing so we async call this function and processing index.ts continues.

Reply:
I think this should have the same effect we want (without the waiting):
```
(async () => {
  await refreshUtxos(bitcoinWallet);
})();
```
If the waiting of `setTimeout` actually has a benefit we can leave it as is, otherwise we could change it.

Reply:
ignore the above comment :D 
```
refreshUtxos(bitcoinWallet);
```
omitting the await does the same trick :)